### ISSUE TITLE
Screen() object hangs forever when trying to init IT8951

### ISSUE THREAD

**Issue by txoof** [txoof/epdlib#12]
When no IT8951 display is connected, the `epdlib.Screen(epd='HD')` hangs forever.

This is related to how the IT8951 module pulls info from the board.
```
IT8951/interface.py in update_system_info(self)
     97         Get information about the system, and store it in class attributes
     98         '''
---> 99         self.spi.write_cmd(Commands.GET_DEV_INFO)
    100         data = self.spi.read_data(20)
    101 
```

**Comment by txoof** [comment]
Related to https://github.com/GregDMeyer/IT8951/issues/36

**Comment by txoof** [comment]
consider wrapping this in a `signal` handler that has a timeout for response:
https://stackoverflow.com/questions/492519/timeout-on-a-function-call

**Comment by aaronr8684** [comment]
@txoof This is something I could fix as part of the rewrite of the Screen class. I don't know enough about the it8951 screens though to know what a reasonable timeout would be. I can research it but what are your thoughts?

**Comment by txoof** [comment]
@aaronr8684 
This is basically a quality-of-life issue for end users. 

Based on my experience, I think a timeout of 15 seconds would be appropriate. I've never had a properly functioning IT8951 take more than 2 seconds to init. 15 seconds should be more than enough time for everything to init if it's working properly.

**Comment by aaronr8684** [comment]
Alright, I'll see about adding it in since we're adding a bunch of new screens. 

Also, I believe the HD screens should be using Omni using combined code if you wanted to test sometime this week. I don't have one to test, so I can't personally confirm.

Do I understand correctly that the "HD"/IT8951 screens are the same thing as SPI/USB/I80 on waveshare's website?

**Comment by txoof** [comment]
Yeah. HD:IT8951:UART/SPI are all synonyms for the same thing. Though there are some oddball boards mixed in there that look like they use HDMI.

**Comment by aaronr8684** [comment]
Yea I saw a few of the HDMI boards from Good Display (I think).

So does that mean the documentation listed on your various epd repos incorrectly say that the SPI/USB/I80 screens aren't supported?

**Comment by txoof** [comment]
> So does that mean the documentation listed on your various epd repos incorrectly say that the SPI/USB/I80 screens aren't supported?

Definitely should indicate that EPDLib supports Waveshare HD screenns. Which repos look wrong? I'll update.

**Comment by aaronr8684** [comment]
On epd_display and PaperPi (last line):
![image](https://user-images.githubusercontent.com/57102156/157084520-616011ec-2f7e-44a4-985a-f5c090138513.png)


Also I believe this statement about the BCM driver in the epdlib readme is no longer true (based on the [v0.1.0 release notes](https://github.com/GregDMeyer/IT8951#updates-for-version-010) ):
![image](https://user-images.githubusercontent.com/57102156/157084967-abaa2f3a-ff95-440c-a7b6-d8b8c009859e.png)

**Comment by txoof** [comment]
I'll tidy those. Thanks for catching.

**Comment by txoof** [comment]
This should be resolved now via [I#36](https://github.com/GregDMeyer/IT8951/issues/36) being closed on the IT8951 driver. Need to verify logic.
